deployment/flatpak/flatpak-usage.md

```diff
@@ -18,7 +18,11 @@ To add the Flathub repository, open the Terminal, and run:
 $ flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
 ```
 
+To add the Fedora Flatpaks repository (Flatpak apps built in the Fedora infrastructure), run:
```

Comment:
```suggestion
To add the Fedora Flatpaks repository (built in Fedora):
```

Reply:
Thanks! I reworded the line as suggested.